haystack/utils/experiment_tracking.py

```diff
@@ -232,6 +230,9 @@ def get_or_create_env_meta_data() -> Dict[str, Any]:
     """
     Collects meta data about the setup that is used with Haystack, such as: operating system, python version, Haystack version, transformers version, pytorch version, number of GPUs, execution environment, and the value stored in the env variable HAYSTACK_EXECUTION_CONTEXT.
     """
+    import torch
```

Comment:
Why not have a try catch here too and show an error message with `pip install transformers[torch]` if the import fails?

Reply:
Very good question and now I can't remember why I did it this way. I reckon there must have been _some_ reason... I'll try to use LazyImport here and see if anything happens.

Reply:
No reason it seems. Fixed! :smile: